server/docker-compose.yml

```diff
@@ -16,7 +16,7 @@ services:
       - "3000:3000"
     depends_on:
       - db
-    command: ["npm", "run", "dev"]
+    command: [ "npm", "run", "dev" ]
```

Comment:
Why were spaces added?

Reply:
It kept auto formatting for me whenever I tried to change it back and save